brainfuck/commands.go

```diff
@@ -56,3 +56,12 @@ func (l loop) execute(mem *memory) {
 		}
 	}
 }
+
+// Increments current memory buffer's cell by five
+type incrementByFive struct{}
```

Comment:
`incrementByFive increments current memory buffer's cell by five`

pls fix it and read how to write docs in golang

Reply:
Fixed